### PR TITLE
S1 typography, detail tokens update

### PR DESCRIPTION
**Pull request by larz0**
## Description

Updated for desktop and mobile:
- detail-cjk-strong-font-weight
- detail-cjk-emphasized-font-weight
- detail-cjk-strong-emphasized-font-weight

Added for desktop and mobile:
- all detail-cjk-size-* tokens, from XL to S

## Motivation and context

These changes are for CJK typography improvements.

## Related issue

[SDS-13798](https://jira.corp.adobe.com/browse/SDS-13798)

## Types of changes

<li> [ ] Patch (bug fixes, typos, mistakes; non-breaking change which fixes an issue) </li>
<li> [x] Minor (add a new token, changing a value, deprecating a token; non-breaking change which adds functionality) </li>
<li> [ ] Major (deleting a token, changing token value type, renaming a token by deprecating the old one; fix or feature that would cause existing functionality to change) </li>

## Checklist:

<li> [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html). </li>
<li> [x] I updated the token in all applicable sets. This applies if updating, adding, or deleting a token that has data across different sets (for example, if the value differs across color themes.) </li>

### CODE REVIEW

**Comment by mrcjhicks** [comment]
**[TESTING spectrum](https://github.com/adobe/spectrum-tokens-studio-data/actions/runs/10585577107)**

READING spectrum-tokens: `/spectrum-tokens/packages/tokens/src/`
READING   tokens studio: `/spectrum-tokens-studio-data/src/tokens-studio/spectrum-colors/`
READING   tokens studio: `/spectrum-tokens-studio-data/src/tokens-studio/spectrum-non-colors/`

  > WARNING - studio token uuid collision: (ab476eec-b592-4890-af8f-74de808cb87f)
  >           spectrum/typography.component/desktop,detail-size-xl
  >           spectrum/typography.component/mobile,detail-size-xl
  >           spectrum/typography.component/mobile,detail-cjk-size-xl
  >           spectrum/typography.component/mobile,detail-cjk-size-l
  >           spectrum/typography.component/mobile,detail-cjk-size-m
  >           spectrum/typography.component/mobile,detail-cjk-size-s
  > WARNING - STUDIO MISSING MODE DEFINITION FOR spectrum/typography.component/detail-cjk-size-xl: ["mobile"]
  > WARNING - STUDIO MISSING MODE DEFINITION FOR spectrum/typography.component/detail-cjk-size-l: ["mobile"]
  > WARNING - STUDIO MISSING MODE DEFINITION FOR spectrum/typography.component/detail-cjk-size-m: ["mobile"]
  > WARNING - STUDIO MISSING MODE DEFINITION FOR spectrum/typography.component/detail-cjk-size-s: ["mobile"]


<details><summary>STUDIO SYNC SUMMARY</summary><p>

```
UPDATED METADATA - SPECTRUM UUID: spectrum/typography.component/desktop.detail-cjk-size-xl
UPDATED METADATA - SPECTRUM UUID: spectrum/typography.component/desktop.detail-cjk-size-l
UPDATED METADATA - SPECTRUM UUID: spectrum/typography.component/desktop.detail-cjk-size-m
UPDATED METADATA - SPECTRUM UUID: spectrum/typography.component/desktop.detail-cjk-size-s
```
</p></details>

<details><summary>SPECTRUM SYNC SUMMARY</summary><p>

```
NEW TOKENS: (4) detail-cjk-size-xl/desktop, detail-cjk-size-l/desktop, detail-cjk-size-m/desktop, detail-cjk-size-s/desktop
CHANGED VALUES: (3) detail-cjk-emphasized-font-weight, detail-cjk-strong-emphasized-font-weight, detail-cjk-strong-font-weight
```
</p></details>

TEST COMPLETE

**Comment by larz0** [comment]
Note: #177 must be merged first in order for `detail-cjk-size-s` to reference the correct value

**Comment by mrcjhicks** [comment]
**[TESTING spectrum](https://github.com/adobe/spectrum-tokens-studio-data/actions/runs/10618787414)**

READING spectrum-tokens: `/spectrum-tokens/packages/tokens/src/`
READING   tokens studio: `/spectrum-tokens-studio-data/src/tokens-studio/spectrum-colors/`
READING   tokens studio: `/spectrum-tokens-studio-data/src/tokens-studio/spectrum-non-colors/`

  > WARNING - studio token uuid collision: (ab476eec-b592-4890-af8f-74de808cb87f)
  >           spectrum/typography.component/desktop,detail-size-xl
  >           spectrum/typography.component/mobile,detail-size-xl
  >           spectrum/typography.component/mobile,detail-cjk-size-xl
  >           spectrum/typography.component/mobile,detail-cjk-size-l
  >           spectrum/typography.component/mobile,detail-cjk-size-m
  >           spectrum/typography.component/mobile,detail-cjk-size-s
  > WARNING - STUDIO MISSING MODE DEFINITION FOR spectrum/typography.component/detail-cjk-size-xl: ["mobile"]
  > WARNING - STUDIO MISSING MODE DEFINITION FOR spectrum/typography.component/detail-cjk-size-l: ["mobile"]
  > WARNING - STUDIO MISSING MODE DEFINITION FOR spectrum/typography.component/detail-cjk-size-m: ["mobile"]
  > WARNING - STUDIO MISSING MODE DEFINITION FOR spectrum/typography.component/detail-cjk-size-s: ["mobile"]


<details><summary>STUDIO SYNC SUMMARY</summary><p>

```
UPDATED METADATA - SPECTRUM UUID: spectrum/typography.letter-spacing
UPDATED METADATA - SPECTRUM UUID: spectrum/typography/desktop.font-size-25
UPDATED METADATA - SPECTRUM UUID: spectrum/typography/mobile.font-size-25
UPDATED METADATA - SPECTRUM UUID: spectrum/typography.component/desktop.detail-cjk-size-xl
UPDATED METADATA - SPECTRUM UUID: spectrum/typography.component/desktop.detail-cjk-size-l
UPDATED METADATA - SPECTRUM UUID: spectrum/typography.component/desktop.detail-cjk-size-m
UPDATED METADATA - SPECTRUM UUID: spectrum/typography.component/desktop.detail-cjk-size-s
```
</p></details>

<details><summary>SPECTRUM SYNC SUMMARY</summary><p>

```
NEW TOKENS: (7) letter-spacing, font-size-25/desktop, font-size-25/mobile, detail-cjk-size-xl/desktop, detail-cjk-size-l/desktop, detail-cjk-size-m/desktop, detail-cjk-size-s/desktop
CHANGED VALUES: (4) cjk-letter-spacing, detail-cjk-emphasized-font-weight, detail-cjk-strong-emphasized-font-weight, detail-cjk-strong-font-weight
CHANGED SCHEMAS: (1) cjk-letter-spacing
```
</p></details>

TEST COMPLETE

File: src/tokens-studio/spectrum-non-colors/spectrum/typography.component/mobile.json [hunk 1]
```diff
@@ -1239,6 +1239,46 @@
       }
     }
   },
+  "detail-cjk-size-xl": {
+    "value": "{font-size-100}",
+    "type": "fontSizes",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "ab476eec-b592-4890-af8f-74de808cb87f",
```
Comment (karstens) [inline src/tokens-studio/spectrum-non-colors/spectrum/typography.component/mobile.json:1247]: @larz0 not sure, where you got these uuids, but if its a complete new token in s1 (and also is a new token for s2), please remove the UUID value. Like you did in the desktop data.

File: src/tokens-studio/spectrum-non-colors/spectrum/typography.component/mobile.json [hunk 2]
```diff
@@ -1239,6 +1239,46 @@
...
+        "name": "detail-size-xl"
+      }
+    }
+  },
+  "detail-cjk-size-l": {
+    "value": "{font-size-75}",
+    "type": "fontSizes",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "ab476eec-b592-4890-af8f-74de808cb87f",
```
Comment (karstens) [inline src/tokens-studio/spectrum-non-colors/spectrum/typography.component/mobile.json:1257]: @larz0 not sure, where you got these uuids, but if its a complete new token in s1 (and also is a new token for s2), please remove the UUID value. Like you did in the desktop data.

File: src/tokens-studio/spectrum-non-colors/spectrum/typography.component/mobile.json [hunk 3]
```diff
@@ -1239,6 +1239,46 @@
...
+        "name": "detail-size-xl"
+      }
+    }
+  },
+  "detail-cjk-size-m": {
+    "value": "{font-size-50}",
+    "type": "fontSizes",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "ab476eec-b592-4890-af8f-74de808cb87f",
```
Comment (karstens) [inline src/tokens-studio/spectrum-non-colors/spectrum/typography.component/mobile.json:1267]: @larz0 not sure, where you got these uuids, but if its a complete new token in s1 (and also is a new token for s2), please remove the UUID value. Like you did in the desktop data.

File: src/tokens-studio/spectrum-non-colors/spectrum/typography.component/mobile.json [hunk 4]
```diff
@@ -1239,6 +1239,46 @@
...
+        "name": "detail-size-xl"
+      }
+    }
+  },
+  "detail-cjk-size-s": {
+    "value": "{font-size-50}",
+    "type": "fontSizes",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "ab476eec-b592-4890-af8f-74de808cb87f",
```
Comment (karstens) [inline src/tokens-studio/spectrum-non-colors/spectrum/typography.component/mobile.json:1277]: @larz0 not sure, where you got these uuids, but if its a complete new token in s1 (and also is a new token for s2), please remove the UUID value. Like you did in the desktop data.

**Comment by mrcjhicks** [comment]
**[TESTING spectrum](https://github.com/adobe/spectrum-tokens-studio-data/actions/runs/10635119161)**

READING spectrum-tokens: `/spectrum-tokens/packages/tokens/src/`
READING   tokens studio: `/spectrum-tokens-studio-data/src/tokens-studio/spectrum-colors/`
READING   tokens studio: `/spectrum-tokens-studio-data/src/tokens-studio/spectrum-non-colors/`

<details><summary>STUDIO SYNC SUMMARY</summary><p>

```
UPDATED METADATA - SPECTRUM UUID: spectrum/typography.letter-spacing
UPDATED METADATA - SPECTRUM UUID: spectrum/typography/desktop.font-size-25
UPDATED METADATA - SPECTRUM UUID: spectrum/typography/mobile.font-size-25
UPDATED METADATA - SPECTRUM UUID: spectrum/typography.component/desktop.detail-cjk-size-xl
UPDATED METADATA - SPECTRUM UUID: spectrum/typography.component/mobile.detail-cjk-size-xl
UPDATED METADATA - SPECTRUM UUID: spectrum/typography.component/desktop.detail-cjk-size-l
UPDATED METADATA - SPECTRUM UUID: spectrum/typography.component/mobile.detail-cjk-size-l
UPDATED METADATA - SPECTRUM UUID: spectrum/typography.component/desktop.detail-cjk-size-m
UPDATED METADATA - SPECTRUM UUID: spectrum/typography.component/mobile.detail-cjk-size-m
UPDATED METADATA - SPECTRUM UUID: spectrum/typography.component/desktop.detail-cjk-size-s
UPDATED METADATA - SPECTRUM UUID: spectrum/typography.component/mobile.detail-cjk-size-s
UPDATED METADATA - SPECTRUM NAME: detail-cjk-size-xl => detail-size-xl
UPDATED METADATA - SPECTRUM NAME: detail-cjk-size-l => detail-size-xl
UPDATED METADATA - SPECTRUM NAME: detail-cjk-size-m => detail-size-xl
UPDATED METADATA - SPECTRUM NAME: detail-cjk-size-s => detail-size-xl
```
</p></details>

<details><summary>SPECTRUM SYNC SUMMARY</summary><p>

```
NEW TOKENS: (11) letter-spacing, font-size-25/desktop, font-size-25/mobile, detail-cjk-size-xl/desktop, detail-cjk-size-xl/mobile, detail-cjk-size-l/desktop, detail-cjk-size-l/mobile, detail-cjk-size-m/desktop, detail-cjk-size-m/mobile, detail-cjk-size-s/desktop, detail-cjk-size-s/mobile
CHANGED VALUES: (10) cjk-letter-spacing, detail-cjk-emphasized-font-weight, detail-cjk-strong-emphasized-font-weight, detail-cjk-strong-font-weight, heading-cjk-heavy-font-weight, heading-cjk-size-s, heading-cjk-size-xs, heading-cjk-size-xxl, heading-cjk-size-xxs, heading-cjk-size-xxxl impacting 12 tokens
CHANGED SCHEMAS: (1) cjk-letter-spacing
```
</p></details>

TEST COMPLETE

**Comment by mrcjhicks** [comment]
**[TESTING spectrum](https://github.com/adobe/spectrum-tokens-studio-data/actions/runs/10635186906)**

READING spectrum-tokens: `/spectrum-tokens/packages/tokens/src/`
READING   tokens studio: `/spectrum-tokens-studio-data/src/tokens-studio/spectrum-colors/`
READING   tokens studio: `/spectrum-tokens-studio-data/src/tokens-studio/spectrum-non-colors/`

<details><summary>STUDIO SYNC SUMMARY</summary><p>

```
UPDATED METADATA - SPECTRUM UUID: spectrum/typography.letter-spacing
UPDATED METADATA - SPECTRUM UUID: spectrum/typography/desktop.font-size-25
UPDATED METADATA - SPECTRUM UUID: spectrum/typography/mobile.font-size-25
UPDATED METADATA - SPECTRUM UUID: spectrum/typography.component.detail-cjk-size-xl
UPDATED METADATA - SPECTRUM UUID: spectrum/typography.component.detail-cjk-size-l
UPDATED METADATA - SPECTRUM UUID: spectrum/typography.component.detail-cjk-size-m
UPDATED METADATA - SPECTRUM UUID: spectrum/typography.component/desktop.detail-cjk-size-s
UPDATED METADATA - SPECTRUM UUID: spectrum/typography.component/mobile.detail-cjk-size-s
```
</p></details>

<details><summary>SPECTRUM SYNC SUMMARY</summary><p>

```
NEW TOKENS: (8) letter-spacing, font-size-25/desktop, font-size-25/mobile, detail-cjk-size-xl, detail-cjk-size-l, detail-cjk-size-m, detail-cjk-size-s/desktop, detail-cjk-size-s/mobile
CHANGED VALUES: (10) cjk-letter-spacing, detail-cjk-emphasized-font-weight, detail-cjk-strong-emphasized-font-weight, detail-cjk-strong-font-weight, heading-cjk-heavy-font-weight, heading-cjk-size-s, heading-cjk-size-xs, heading-cjk-size-xxl, heading-cjk-size-xxs, heading-cjk-size-xxxl impacting 12 tokens
CHANGED SCHEMAS: (1) cjk-letter-spacing
```
</p></details>

TEST COMPLETE

**Comment by larz0** [comment]
@karstens I've removed UUIDs and corrected names.

**Comment by karstens** [comment]
> @karstens I've removed UUIDs and corrected names.

Thank you @larz0! This is good to go. Just need another designer to review and approve. 👍

File: src/tokens-studio/spectrum-non-colors/spectrum/typography.component/mobile.json [hunk 5]
```diff
@@ -1239,6 +1239,46 @@
...
+    "type": "fontSizes",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "detail-cjk-size-m"
+      }
+    }
+  },
+  "detail-cjk-size-s": {
+    "value": "{font-size-50}",
```
Comment (lynnhao) [inline src/tokens-studio/spectrum-non-colors/spectrum/typography.component/mobile.json:1273]: I think this one should be `font-size-25`

Comment (larz0) [reply src/tokens-studio/spectrum-non-colors/spectrum/typography.component/mobile.json:1273]: Thanks Lynn, looks like Tokens Studio might automatically apply the next available token if we explicitly add a non-existent token as value in JSON 😅

Comment (larz0) [reply src/tokens-studio/spectrum-non-colors/spectrum/typography.component/mobile.json]: Fixed 👍

**Comment by mrcjhicks** [comment]
**[TESTING spectrum](https://github.com/adobe/spectrum-tokens-studio-data/actions/runs/10638540995)**

READING spectrum-tokens: `/spectrum-tokens/packages/tokens/src/`
READING   tokens studio: `/spectrum-tokens-studio-data/src/tokens-studio/spectrum-colors/`
READING   tokens studio: `/spectrum-tokens-studio-data/src/tokens-studio/spectrum-non-colors/`

<details><summary>STUDIO SYNC SUMMARY</summary><p>

```
UPDATED METADATA - SPECTRUM UUID: spectrum/typography.letter-spacing
UPDATED METADATA - SPECTRUM UUID: spectrum/typography/desktop.font-size-25
UPDATED METADATA - SPECTRUM UUID: spectrum/typography/mobile.font-size-25
UPDATED METADATA - SPECTRUM UUID: spectrum/typography.component.detail-cjk-size-xl
UPDATED METADATA - SPECTRUM UUID: spectrum/typography.component.detail-cjk-size-l
UPDATED METADATA - SPECTRUM UUID: spectrum/typography.component.detail-cjk-size-m
UPDATED METADATA - SPECTRUM UUID: spectrum/typography.component.detail-cjk-size-s
```
</p></details>

<details><summary>SPECTRUM SYNC SUMMARY</summary><p>

```
NEW TOKENS: (7) letter-spacing, font-size-25/desktop, font-size-25/mobile, detail-cjk-size-xl, detail-cjk-size-l, detail-cjk-size-m, detail-cjk-size-s
CHANGED VALUES: (10) cjk-letter-spacing, detail-cjk-emphasized-font-weight, detail-cjk-strong-emphasized-font-weight, detail-cjk-strong-font-weight, heading-cjk-heavy-font-weight, heading-cjk-size-s, heading-cjk-size-xs, heading-cjk-size-xxl, heading-cjk-size-xxs, heading-cjk-size-xxxl impacting 12 tokens
CHANGED SCHEMAS: (1) cjk-letter-spacing
```
</p></details>

TEST COMPLETE

**Review by lynnhao — APPROVED** [review]
Looks good now, thank you!

**Pull request merged**